include/orwell/Server.hpp

```diff
@@ -37,6 +38,7 @@ class Server : public IServer
 			std::string const & iAgentUrl =  "tcp://*:9003",
 			std::string const & iPullUrl = "tcp://*:9000",
 			std::string const & iPublishUrl = "tcp://*:9001",
+			std::string const & iReplyUrl = "tcp://*:9002",
```

Comment:
I am not really happy with the name "replier", but do not have a better alternative in mind.

Reply:
Me neither, it should be fine.